independent-publisher-connectors/Proposal - GIPHY (Independent Publisher)/apiDefinition.swagger.json

```diff
@@ -0,0 +1,848 @@
+                    "profile_url": {
+                      "type": "string",
+                      "description": "profile_url"
+                    },
+                    "username": {
+                      "type": "string",
+                      "description": "username"
+                    },
+                    "display_name": {
+                      "type": "string",
+                      "description": "display_name"
+                    },
+                    "description": {
+                      "type": "string",
+                      "description": "description"
+                    },
+                    "instagram_url": {
+                      "type": "string",
+                      "description": "instagram_url"
+                    },
+                    "website_url": {
+                      "type": "string",
+                      "description": "website_url"
+                    },
+                    "is_verified": {
+                      "type": "boolean",
+                      "description": "is_verified"
+                    }
+                  },
+                  "description": "user"
+                },
+                "analytics_response_payload": {
+                  "type": "string",
+                  "description": "analytics_response_payload"
+                },
+                "analytics": {
+                  "type": "object",
+                  "properties": {
+                    "onload": {
+                      "type": "object",
+                      "properties": {
+                        "url": {
+                          "type": "string",
+                          "description": "url"
+                        }
+                      },
+                      "description": "onload"
+                    },
+                    "onclick": {
+                      "type": "object",
+                      "properties": {
+                        "url": {
+                          "type": "string",
+                          "description": "url"
+                        }
+                      },
+                      "description": "onclick"
+                    },
+                    "onsent": {
+                      "type": "object",
+                      "properties": {
+                        "url": {
+                          "type": "string",
+                          "description": "url"
+                        }
+                      },
+                      "description": "onsent"
+                    }
+                  },
+                  "description": "analytics"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get GIF",
+        "description": "Returns a list of GIFs related to the search string provided as input",
+        "operationId": "Get-GIF",
+        "parameters": [
+          {
+            "name": "API_KEY",
+            "in": "query",
+            "required": true,
+            "type": "string"

```

Comment:
@priyanshu870 please add 
"description": "Your description", could be "Enter your API Key."
"x-ms-summary": "Your Summary", could be "APIKey"

Reply:
Done